recipe/0005-Add-boost-throw_exception-implementation.patch

```diff
@@ -0,0 +1,32 @@
+From 497878f223a729d8458f7abf4a861b61bb487ea8 Mon Sep 17 00:00:00 2001
+From: "Uwe L. Korn" <uwe.korn@quantco.com>
+Date: Mon, 17 May 2021 16:26:53 +0200
+Subject: [PATCH 5/5] Add boost::throw_exception implementation
```

Comment:
Is there an upstream issue or PR for this?

Reply:
No, I haven't found one. This patch is a new addition that fixes a linking issue that only occurs on Windows if you dynamically link `libLIEF.dll`. This is only supported using the previously applied patches. We should probably make a pull request that upstreams all patches as one pull request.